src/Management/src/Endpoint/ConfigurationSchema.json

```diff
@@ -418,13 +421,27 @@
                   "type": "string",
                   "description": "Gets or sets the relative path at which this endpoint is exposed."
                 },
+                "RequestHeaders": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "Gets request header values that are allowed to be logged.\n\nIf a request header is not present in the , the header name will be logged with a redacted value. Request headers can contain authentication tokens, or private information which may have regulatory concerns under GDPR and other laws. Arbitrary request headers should not be logged unless logs are secure and access controlled and the privacy impact assessed."
```

Comment:
> If a request header is not present in the , the header

Looks like the `cref` doesn't properly resolve. Same issue in `ResponseHeaders` below.

Reply:
Fixed by https://github.com/SteeltoeOSS/Steeltoe/pull/1358.

Reply:
Rebased on that branch